ui/src/dashboards/components/CellEditorOverlay.tsx

```diff
@@ -224,9 +227,11 @@ class CellEditorOverlay extends Component<Props, State> {
     let queries: CellQuery[] = queryDrafts
 
     if (this.isFluxQuery) {
+      onChangeScript(draftScript)
```

Comment:
I think we can omit this line, since `collectCell` gets called as the CEO is closing anyway. Keeping it there will fire off another set of queries for the CEO visualization, but they aren't needed since the CEO is about to be hidden.

Reply:
if we omit this line, then when we return to the dashboard, it shows the old cell instead of the new changes

Reply:
fixed 👍